crates/bevy_state/src/state/freely_mutable_state.rs

```diff
@@ -9,6 +9,11 @@ use super::transitions::*;
 ///
 /// While ordinary states are freely mutable (and implement this trait as part of their derive macro),
 /// computed states are not: instead, they can *only* change when the states that drive them do.
+#[diagnostic::on_unimplemented(
+    message = "`{Self}` is not a `FreelyMutableState`",
+    label = "invalid `FreelyMutableState`",
```

Comment:
"is not freely mutable"?

Perhaps change the label to "does not implement `FreelyMutableState`", if you make this change

Reply:
The issue here is `NextState` is only one of the methods you wont have access to (e.g., `App::init_state(...)`). What I'll do here is remove the `message` and `label`, leaving only the `help` message pointing to `#[derive(States)]`.